src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/CompleteConfigurationTest.php

```diff
@@ -87,6 +87,14 @@ public function testFirewalls()
                 'security.user_checker',
                 '.security.request_matcher.xmi9dcw',
                 false,
+                false,
+                '',
+                '',
+                '',
+                '',
+                '',
+                [],
+                null
```

Comment:
What's asserted here are the constructor arguments for the firewall's `FirewallConfig`. Now that `ResolveChildDefinitionsPass` is executed in the test, the default arguments provided by the `ChildDefiniton`s parent are propagated.

So I guess we just add them to the assert?

Reply:
yes 👍

Reply:
Good, that's what I did :)